dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp

```diff
@@ -808,7 +806,7 @@ ModelBus::evalJt(SparseMatrix& jt, const double& /*cj*/, const int& rowOffset) {
     for (; iter != iiDerivativesValues.end(); ++iter) {
       jt.addTerm(iter->first + rowOffset, iter->second);
 #ifdef _DEBUG_
-      Trace::debug("NETWORK") << id_ << " : II_DERIVATIVE[" << iter->first << "] = " << iter->second << Trace::endline;
+      Trace::debug(Trace::NETWORK) << id_ << " : II_DERIVATIVE[" << iter->first << "] = " << iter->second << Trace::endline;
```

Comment:
To be honest, I would delete these ones on the derivatives: very verbose and too much information to really be able to use it from my point of view

Reply:
Done